packages/notebook/style/base.css

```diff
@@ -329,8 +329,8 @@
 }
 
 .jp-mod-sideBySide.jp-Notebook .jp-CodeCell.jp-mod-resizedCell {
-  grid-template-columns: minmax(0, 1fr) min-content minmax(
-      0,
+  grid-template-columns: minmax(70px, 1fr) min-content minmax(
```

Comment:
a quick note- I haven't pushed this upstream yet, but I also had to set this 70px for `jp-CodeCell` minMax (otherwise adjacent cells reacting to the min/max don't respect the minimum value set in `for jp-mod-resizedCell`)